### ISSUE TITLE
BCN3D+ Configuration.h file for Marlin 1.1.9

### ISSUE THREAD

**Issue by philippejadin** [BCN3D/BCN3DPlus-Firmware#11]
Here you can find a working Configuration.h file in order to use the latest marlin firmware with a bcn3d+ printer : 
https://gist.github.com/philippejadin/8b566562940c7e1d0b4f5fe89a2edb96

- Download marlin 1.1.9
- Change Configuratio.h with the one provided in the gist above
- Compile and upload to your printer using arduino ide

**Comment by renaudferte** [comment]
Thanks, i just did that but I have a display glitch. Have you ever encountered such thing ?
Like I can feel the data but really can't navigate through.

**Comment by jbournonville** [comment]
Yes same problem. Just put those lines at the end on the `Configuration.h` to solve the issue.
```
#define ST7920_DELAY_1 DELAY_NS(0) 
#define ST7920_DELAY_2 DELAY_NS(250) 
#define ST7920_DELAY_3 DELAY_NS(250)
```

**Comment by renaudferte** [comment]
Thank you so very much ! it works all right now !
